src/guides/en/understand-devtools/index.md

```diff
@@ -0,0 +1,236 @@
+
+## Who is DevTools for
+
+The vast majority of the people who use DevTools are web developers, but others may find it useful too.
+
+When you're writing the code for a website, chances are that bugs will happen. You might make a mistake in the HTML content, or in the CSS style of the page, or in the JavaScript logic. DevTools can help you investigate and fix those bugs. With DevTools, you can inspect the code that the browser is running to display your page, find what the problem is, and then correct it in your source code.
+
+DevTools also allows you to make changes to the page that's being debugged. For example, if you made a CSS mistake in the color of an element, you can change the element's color to test a fix. Because of this ability to make changes to a page, DevTools might be appealing to non web developers too. For example, you could use DevTools to slightly tweak a web page and make it easier to read: you could remove various elements or change the text size and color.
+
+{% insertTip "remove-annoying-overlays" %}
+
+Continue reading to learn more about how these changes are made, and what they affect.
+
+## How DevTools works
+
+### DevTools debugs the runtime version of your webpage
+
+To understand how DevTools works, you first need to understand how browsers work a little bit. When you enter a URL to a webpage in your browser, the document that lives at that URL, and its dependencies, are fetched and parsed by the browser. The entire code that the webpage needs is run and the page is rendered into the browser tab.
+
+If you're the author of this webpage, the HTML, CSS, and perhaps JavaScript code that you wrote in your text editor is what led to the page being rendered in the browser. But the code was parsed, interpreted, and run by the browser, which resulted in the page being displayed.
+
+What runs in the browser for your page to show up is different than the code you wrote, it's an runtime version of it.
+
+**The runtime version of your webpage is what DevTools debugs. DevTools doesn't debug your original source code.**
+
+For example, when the browser fetches the HTML content of your web page, it parses it and generates a tree of [DOM](https://developer.mozilla.org/docs/Web/API/Document_Object_Model/Introduction) nodes from it. The DOM tree is what is then used to render the page visually in the browser. DevTools let's you see and interact with the DOM tree, not with the original HTML source code.
+
+![Diagram illustrating that your source code, and the runtime version of your code in the browser, are different, and that DevTools can only see the runtime](./source-vs-runtime.png)
+
+### DevTools can't change your source code
+
+Because DevTools debugs the runtime version of your webpage, instead of the original source code, the changes you make in DevTools are not saved to your source code, and are not persisted when you reload the page.
```

Comment:
Local overrides and Workspaces does allow you to persist changes in the original source code. Perhaps this could be counter-intuitive while using DevTools for local development. In principal, the sentence is correct though.